src/SFA.DAS.AssessorService.ApplyTypes/Roatp/GatewayReviewStatus.cs

```diff
@@ -7,5 +7,6 @@ public static class GatewayReviewStatus
         public const string InProgress = "In Progress";
         public const string Approved = "Approved";
         public const string Declined = "Declined";
+        public const string AskForClarification = "Ask Clarification";
```

Comment:
Wording doesn't make sense to be. Should either be:

Ask For Clarification    or    Clarification

Reply:
I will change it to "Clarification" as "Ask For Clarification" goes over the limit of the database field size.